deployer/steps.go

```diff
@@ -12,22 +12,22 @@ import (
 func (d *Deployer) StepPrepNetbootDir() error {
 	return d.Add(opPrepareNetboot, herd.WithCallback(
 		func(ctx context.Context) error {
-			return os.MkdirAll(d.dstNetboot(), 0700)
+			return os.MkdirAll(d.dstNetboot(), 0755)
```

Comment:
we should probably use a constant for this as we do on other places, but lets fix it first